CMakeLists.txt

```diff
@@ -248,7 +248,9 @@ endif()
 # installing nvfuser headers
 install(DIRECTORY "${NVFUSER_SRCS_DIR}/"
       DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/nvfuser"
-      FILES_MATCHING PATTERN "*.h" )
+      FILES_MATCHING
+      PATTERN "*.h"
+      PATTERN "type_traits")
```

Comment:
I think it is fine. Should we be more specific and say `csrc/C++20/type_traits` instead of  `type_traits`?

Reply:
Can we also add this here:
https://github.com/NVIDIA/Fuser/blob/eabfc2b0701e076c8e1530894019cd8c0cd0cf8c/setup.py#L318-L326

We probably need a test coverage on that as well. But let's not worry about that now.